testing.rst

```diff
@@ -321,52 +320,11 @@ concrete one::
         }
     }
 
-In order to make the alias public, you will need to update configuration for
-the ``test`` environment as follows:
+.. deprecated:: 6.3
```

Comment:
This get removed in 7.0

I think we should add a versionadded that this is now possible from 6.3 on

Reply:
Does the first version (https://github.com/symfony/symfony-docs/commit/1f09911bd9901d98d1b29ffe3ef9de5901512b08) what you mean, but with ``versionadded`` instead of ``tip``?

Reply:
Always keep in mind to keep the info itself "forever" and only the `versionadded` directive unit the next major

Reply:
Alright, it gets more and more clear. I tried a new version that will "keep the info forever" but also with a ``versionadded``. Hope it is the way to go!